src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs

```diff
@@ -369,5 +373,62 @@ private void ScrollViewer_PreviewMouseWheel(object sender, MouseWheelEventArgs e
             }
             e.Handled = true;
         }
+
+        private void importTextBlock_MouseLeftButtonDown(object sender, MouseButtonEventArgs e)
+        {
+            string[] fileFilter = { string.Format(Res.FileDialogImportSettingsFiles, "*.xml") };
+            System.Windows.Forms.OpenFileDialog openFileDialog = new System.Windows.Forms.OpenFileDialog();
+            openFileDialog.Filter = String.Join("|", fileFilter);
+            openFileDialog.Title = Res.ImportSettingsDialogTitle;
+            openFileDialog.Multiselect = false;
+            openFileDialog.RestoreDirectory = true;
+
+            System.Windows.Forms.DialogResult result = openFileDialog.ShowDialog();
+            if (result == System.Windows.Forms.DialogResult.OK)
+            {
+                try
+                {
+                    viewModel.importSettings(openFileDialog.FileName);
+                    Wpf.Utilities.MessageBoxService.Show(
+                       Res.ImportSettingsSuccessMessage, Res.ImportSettingsDialogTitle, MessageBoxButton.OK, MessageBoxImage.Information);
+                }
+                catch (LibraryLoadFailedException ex)
```

Comment:
for other types of exceptions, what would happen?

Reply:
It will handling as a generic one.